### PR TITLE
Template Builder with Bearer Auth support

### PR DESCRIPTION
**Pull request by venkateshwarv**
Template Builder and Action Builder compatible to Admin portal.

### CODE REVIEW

File: ActionBuilder/package.json [hunk 1]
```diff
@@ -26,21 +26,22 @@
...
     "lit-element": "2.3.1",
     "lit-html": "1.2.1"
   },
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.0",
-    "@cjaas/common-components": "3.0.0",
-    "@momentum-ui/core": "19.15.4",
-    "@momentum-ui/icons": "8.21.0",
+    "@cjaas/common-components": "file:../../cjaas-common-components",
```
Comment (elenagiere) [inline ActionBuilder/package.json:40]: probably want to make this more of a actual npm version.

File: TemplateBuilder/src/assets/styles/template-builder.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+:host {
+  display: block;
+  padding: 0.75rem;
+  overflow-x: auto;
+  min-width: 1368px;
+  color: var(--md-textColor-secondary);
```
Comment (elenagiere) [inline TemplateBuilder/src/assets/styles/template-builder.scss:13]: this seems like an random color to choose. also, provide fallback color.

File: ActionBuilder/package.json (hunk 1 above)
Comment (venkateshwarv) [reply ActionBuilder/package.json:40]: Yes. Waiting on Common Component PR to be accepted.

File: ActionBuilder/src/assets/styles/action-builder.scss [hunk 3]
```diff
@@ -66,21 +66,20 @@ md-icon {
 
 .summary {
   font-size: 12px;
-  color: #000000;
+  color: var(--md-primary-text-color);
```
[the remark this answers is not in the document]
Comment (venkateshwarv) [reply ActionBuilder/src/assets/styles/action-builder.scss]: Addressed

File: TemplateBuilder/src/assets/styles/template-builder.scss (hunk 2 above)
Comment (venkateshwarv) [reply TemplateBuilder/src/assets/styles/template-builder.scss]: Addressed

**Review by elenagiere — APPROVED** [review]

**Pull request merged**